### PR TITLE
Manual ref doc update from release 1.6 with galley removed.

### PR DESCRIPTION
**Pull request by ericvn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/config/proxy_extensions/stats/index.html [hunk 1]
```diff
@@ -24,7 +24,6 @@ <h2 id="MetricConfig">MetricConfig</h2>
 <th>Field</th>
 <th>Type</th>
```
Comment (mandarjog) [inline content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: @richardwxn can you check why this is `proxy_extensions` and why others like attribute_gen are not?
I would like all proxy_exntensions to be under `reference/config/proxy_extensions`

Comment (richardwxn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: the location tag of newly added proto docs are different: https://github.com/istio/proxy/blob/master/extensions/attributegen/config.proto#L21 I can fix them

Comment (richardwxn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: the accesslog one is correct, so we just need to update the attributegen one

Comment (richardwxn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: @ericvn Would you mind rerun it again after I update the proxy doc

Comment (ericvn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: I can do that. Note that we are pulling from the release-1.6 branch so will need to wait on that.

Comment (richardwxn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: Yep I created a PR for 1.6: https://github.com/istio/proxy/pull/2833

Comment (richardwxn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:25]: will let you know when it is merged

**Comment by ericvn** [comment]
Let's just get this PR in. It should then allow the AUTOMATOR PR to be merged, and then the other changes will come in during the daily chron job.

**Pull request merged**